pandas/core/series.py

```diff
@@ -926,6 +926,18 @@ def _repr_footer(self):
                                                len(self),
                                                str(self.dtype.name))
 
+    def _repr_html_(self, *args, **kwargs):
+        df = self.to_frame()
+        if self.name is None:
+            df.columns = ['']
+        return df._repr_html_(*args, **kwargs)
+
+    def to_html(self, *args, **kwargs):
+        df = self.to_frame()
```

Comment:
I have called DataFrame's _repr_html in Series _repr_html and DataFrame's to_html in Series to_html.

If you look at the test case they have 2 different behaviours/outcomes.

Do you recommend only calling _repr_html in both cases and the behaviour being inconsistent with the DataFrame behaviour?